src/history/hs_rec.c

```diff
@@ -1046,3 +1046,103 @@ __hs_delete_reinsert_from_pos(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor, ui
 
     return (ret);
 }
+
+/*
+ * __hs_delete_record --
+ *     Delete the update left in the history store
```

Comment:
Can we reword to:
"Delete an update from the history store"? We can probably add information about the case when we don't have to delete anything.